src/azure-cli/HISTORY.rst

```diff
@@ -59,17 +59,17 @@ Release History
 
 * Document allowed values for sql db create --sample-name
 
-**VM**
+**Compute**
```

Comment:
For consistency with previous release notes, this should be called "VM": https://docs.microsoft.com/en-us/cli/azure/release-notes-azure-cli?view=azure-cli-latest#vm